style.css

```
@@ -292,6 +292,112 @@ section.works {
   gap: 0.66rem;
 }
 
+/* contact */
+.contact {
+  background-color: #6070ff;
+  margin-top: 15.2rem;
+  border-top-left-radius: 7.33rem;
+  padding: 20px 0 100px 0;
+}
+
+.contact div {
+  padding: 7.6rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.8rem;
+  background-image: url("/images/contact-form-shapes-mobile.svg");
```

- [ ] The background image is not yet displaying, I observed that you did not reference the source file properly. Perhaps, you can try any of the two options below:
```suggestion
  background-image: url("./images/contact-form-shapes-mobile.svg");
```

```suggestion
  background-image: url("images/contact-form-shapes-mobile.svg");
```